server/public/styles.css

```diff
@@ -0,0 +1,19 @@
+.landing-page-image {
```

Comment:
In my styles.css I had to add a 
`body {
margin: 0;
} `

To get rid of the white space around it. I view the app with the iPhone mock around it. 
![Screen Shot 2020-05-05 at 12 15 38 PM](https://user-images.githubusercontent.com/59104550/81106284-2c89fc80-8eca-11ea-82d7-83686badf3eb.png)